project/Dependencies.scala

```diff
@@ -32,7 +32,7 @@ object Dependencies {
     scalaTestVersion := "3.0.4",
     java8CompatVersion := {
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, n)) if n >= 12 ⇒ "0.8.0"
+        case Some((2, n)) if n >= 12 ⇒ "0.9.0"
```

Comment:
is it binary compatible with 0.8, which is a requirement if we are to change it in a patch version

is this needed for Scala 2.13, then another case n >= 13 can be added

Reply:
Hmm indeed I'm not so confident - good idea on the `case n >= 13`, will try that!

(we should remember to move to 0.9.0 when we can break bincompat, but we'll go through library updates at that point anyway)